sdk/storage/Azure.Storage.Blobs/tests/ServiceClientTests.cs

```diff
@@ -694,6 +694,129 @@ public async Task UndeleteBlobContainerAsync_Error()
                 e => Assert.AreEqual(BlobErrorCode.ContainerNotFound.ToString(), e.ErrorCode));
         }
 
+        [Test]
+        [ServiceVersion(Min = BlobClientOptions.ServiceVersion.V2020_06_12)]
+        public async Task RenameBlobContainerAsync()
+        {
+            // Arrange
+            BlobServiceClient service = GetServiceClient_SharedKey();
+            string oldContainerName = GetNewContainerName();
+            string newContainerName = GetNewContainerName();
+            BlobContainerClient container = InstrumentClient(service.GetBlobContainerClient(oldContainerName));
+            await container.CreateAsync();
+
+            // Act
+            BlobContainerClient newContainer = await service.RenameBlobContainerAsync(
+                destinationContainerName: newContainerName,
+                sourceContainerName: oldContainerName);
+
+            // Assert
+            await newContainer.GetPropertiesAsync();
+
+            // Cleanup
+            await newContainer.DeleteAsync();
+        }
+
+        [Test]
+        [ServiceVersion(Min = BlobClientOptions.ServiceVersion.V2020_06_12)]
+        public async Task RenameBlobContainerAsync_AccountSas()
```

Comment:
Note that accounting to the spec, this API works with shared key, OAuth, and Account SAS.  It specifically mentions it does not work with User Delegation SAS, and Service SAS was not mentioned.